companion/src/simulation/telemetrysimu.h

```diff
@@ -27,6 +27,7 @@
 #include <QDateTime>
 #include <QtCore/qmath.h>
 #include <QFileDialog>
+#include <QRegularExpression>
```

Comment:
Any particular reason to put this in the header file?  Since there's no QRegularExpression declared in the header, this would probably make more sense in the cpp file.  Not a big deal either way.